### DISCUSSION TITLE
GitLab

### DISCUSSION THREAD

**Discussion by wizard-28** [catppuccin/catppuccin#1833] [Port Requests]

**Comment by NamesCode** [comment]
Yup!
Working on it soon :)

**Comment by sgoudham** [comment]
Closing as further discussion should take place here: https://github.com/catppuccin/catppuccin/issues/2232

**Comment by sgoudham** [comment]
Reopening due to comments/concerns shared in https://github.com/catppuccin/catppuccin/issues/2232#issuecomment-1897588179
The port should now be implemented as a userstyle in [catppuccin/userstyles](https://github.com/catppuccin/userstyles)
